E-Commerce/readme.md

```diff
@@ -45,13 +45,81 @@ erDiagram
 
 Write SQL queries to complete the following tasks:
 
-- [ ] List all the products whose name contains the word "socks"
-- [ ] List all the products which cost more than 100 showing product id, name, unit price, and supplier id
-- [ ] List the 5 most expensive products
+- [x] List all the products whose name contains the word "socks"
+
+```psql
+SELECT * FROM products
+WHERE product_name LIKE '%socks%'
+```
+
+- [x] List all the products which cost more than 100 showing product id, name, unit price, and supplier id
+
+```psql
+SELECT p1.prod_id, p1.supp_id, p1.unit_price, p2.product_name FROM product_availability p1
+JOIN products p2
+ON p1.prod_id = p2.id
+WHERE unit_price >= 100
+```
+
+- [x] List the 5 most expensive products
+
+```psql
+SELECT * FROM product_availability
+WHERE unit_price >= 10
```

Comment:
What's the reason for this WHERE command? Can you please explain?

Reply:
it was for training and testing and now solve it